### ISSUE TITLE
Add copy to clipboard button on `:hover` over usage examples

### ISSUE THREAD

**Issue by danielbayley** [tldr-pages/tldr#8115]
Similar to how [fenced code blocks](https://github.com/tldr-pages/tldr#how-do-i-use-it) work right here on GitHub…

**Comment by navarroaxel** [comment]
@danielbayley Are you talking about https://tldr.sh?

**Comment by danielbayley** [comment]
> @danielbayley Are you talking about https://tldr.sh?

@navarroaxel Yes, or more accurately https://tldr.ostera.io

**Comment by navarroaxel** [comment]
yep, your ticket should be submitted here: https://github.com/ostera/tldr.jsx
